platform/commonUI/general/src/directives/MCTSplitPane.js

```diff
@@ -218,20 +221,21 @@ define(
                 $scope.$on('$destroy', function () {
                     $interval.cancel(activeInterval);
                 });
-
+                
 
                 // Interface exposed by controller, for mct-splitter to user
                 return {
                     anchor: function () {
                         return anchor;
                     },
-                    position: function (value) {
-                        if (arguments.length > 0) {
-                            setUserWidthPreference(value);
-                            return getSetPosition(value);
-                        } else {
+                    position: function (initialValue, newValue) {
```

Comment:
I'm unclear on why this new `initialValue` argument was introduced; I only see it used in specs.

Reply:
this is to make sure the div sizes and splitter position were not being updated when only clicked and not dragged, I added the missing part on mctSplitter

Reply:
Ah, got it, I see.

I think it would be simpler to do that `initialValue !== newValue` check from MCTSplitter, in that case, since it's where that input is actually being handled. Keeps the interface to MCTSplitPane's controller just a little simpler (getter-setter for position)